plugins/ilert/src/hooks/useAlerts.ts

```diff
@@ -38,30 +32,30 @@ export const useIncidents = (
     page: 0,
     pageSize: 10,
   });
-  const [states, setStates] = React.useState<IncidentStatus[]>([
+  const [states, setStates] = React.useState<AlertStatus[]>([
     ACCEPTED,
     PENDING,
   ]);
-  const [incidentsList, setIncidentsList] = React.useState<Incident[]>([]);
-  const [incidentsCount, setIncidentsCount] = React.useState(0);
+  const [alertsList, setAlertsList] = React.useState<Alert[]>([]);
+  const [alertsCount, setAlertsCount] = React.useState(0);
   const [isLoading, setIsLoading] = React.useState(false);
 
-  const fetchIncidentsCall = async () => {
+  const fetchAlertsCall = async () => {
```

Comment:
I know it's not part of this pull request to address, but it's a bit odd how this uses useAsync AND ALSO manual handling of the resulting values and loading states. Normally you do like `const { value, error, loading } = useAsync(...` and no extra useState at all.

Reply:
Thanks for the advice, i would have to put in more time to refactor code here, most likely will be something which will be done with a future update!